templates/confirmation/confirm_remote_server.html

```diff
@@ -0,0 +1,49 @@
+{% extends "zerver/portico_signup.html" %}
+
+{% block portico_content %}
+<div class="app portico-page">
+    <div class="app-main portico-page-container center-block flex full-page account-creation new-style">
+        <div class="inline-block">
+            <div class="bg-image"></div>
+
+            <div class="white-box">
+                {% if confirmed %}
+                    <p>Your remote server, {{status_obj.hostname}}, has successfully registered
+                        with us. You can now start sending push notifications.
+                    </p>
+                {% else %}
+
+                    <p class="lead">Whoops, something's not right. We couldn't find your confirmation ID!</p>
+
+                    {% if verbose_support_offers %}
```

Comment:
We're mostly getting rid of verbose_support_offers, so I think we can just keep the "verbose_support_offers=True" branch of this